samples/broker/dlq/README.md

```diff
@@ -0,0 +1,246 @@
+# RabbitMQ Knative Eventing Broker DLQ Example
+
+## Prerequisites
+
+install the Broker as per [here](../../../broker/README.md)
+
+## Overview
+
+What this demo shows is how the Dead Letter Queue works with the RabbitMQ using
+a very simple example. It demonstrates how failed events get sent to Dead Letter
+Sink while successfully processed events do not.
+
+## Components
+
+* [failer](../../../cmd/failer/main.go) is a function which takes in a CloudEvent
+  and depending on what the specified HTTP response code in the message data is
+  will respond with that. So, to simulate a failure, we just send it a CloudEvent
+  with a payload of 500 and it's going to simulate a failure, by default it will
+  respond with a 200, hence indicating that it processed the event successfully
+  and it should be considered handled.
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
- [failer](../../../cmd/failer/main.go) is a function which takes in a
  CloudEvent and depending on what the specified HTTP response code in the
  message data is will respond with that. So, to simulate a failure, we just
  send it a CloudEvent with a payload of 500 and it's going to simulate a
  failure, by default it will respond with a 200, hence indicating that it
  processed the event successfully and it should be considered handled.
```